### PR TITLE
Single post - amenities, social networks and contact footer

### PR DESCRIPTION
**Pull request by GBD95**
Added amenities, social networks and contact footer, need to update design

### CODE REVIEW

File: components/Hotel/Amenities.vue [hunk 1]
```diff
@@ -4,17 +4,30 @@
 
     <div class="mt-5 flex flex-col flex-wrap gap-3">
       <div
-        class="flexitems-center justify-center"
-        v-for="data in amenities"
+        class="flex items-center justify-start"
+        v-for="(data, index) in visibleAmenities"
```
Comment (LemiKg) [inline components/Hotel/Amenities.vue:8]: Cemu sluzi ovaj index?

File: components/Hotel/Amenities.vue [hunk 2]
```diff
@@ -25,9 +38,27 @@ import { Amenity } from "types/hotel";
...
 const backendUrl = runtimeConfig.public.backendUrl;
 
-defineProps<{
+const props = defineProps<{
   amenities: Array<{
     amenity: Amenity;
   }>;
 }>();
+
+// Display only 5 amenities initially
```
Comment (LemiKg) [inline components/Hotel/Amenities.vue:47]: Ovi komentari nam ne trebaju

**Review by LemiKg — CHANGES_REQUESTED** [review]

File: components/Hotel/SocialNetworks/Icon.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<template>
+  <a
+    class="flex items-center justify-center p-2"
+    :href="`${link}`"
```
[the remark this answers is not in the document]
Comment (GBD95) [reply components/Hotel/SocialNetworks/Icon.vue:4]: Pa onda nas vodi sa naseg sajta, valjda nam to nije cilj nego da otvori novi tab.

File: components/Hotel/Amenities.vue (hunk 1 above)
Comment (GBD95) [reply components/Hotel/Amenities.vue:8]: the index must not be an obstacle, obrisao index i :key="index"

File: components/Hotel/Amenities.vue (hunk 2 above)
Comment (GBD95) [reply components/Hotel/Amenities.vue:47]: Obrisano

File: components/Hotel/SocialNetworks/Icon.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (LemiKg) [reply components/Hotel/SocialNetworks/Icon.vue:4]: Ne pricam za target atribut nego za :href
Ne treba ti `:href="`${link}`"` nego `:href="link"`

**Pull request merged**